test/integration/dockerregistryclient_test.go

```diff
@@ -224,14 +222,19 @@ func doTestRegistryClientImage(t *testing.T, registry, version string) {
 	}
 }
 
+func TestRegistryClientImageV2ManifestV2(t *testing.T) {
+	t.Skip("openshift/schema-v2-test-repo is being pushed by Docker 1.10, which causes this test to fail because digests don't match")
```

Comment:
It's broken for the same reason as #9130.  Nothing has changed in this. If it worked, then initially the test would not break.  I assume that our client does not fully support v2 schema. I think this is related to https://github.com/fsouza/go-dockerclient/issues/488